src/main/component/Editor.svelte

```diff
@@ -117,10 +134,11 @@
       ...conf,
       target: element,
       inline: inline !== undefined ? inline : conf.inline !== undefined ? conf.inline : false,
-      readonly: disabled,
       license_key: licenseKey,
       setup: (editor: TinyMCEEditor) => {
         editorRef = editor;
+        setDisabled(editorRef, disabled, readonly);
+        setReadonly(editorRef, readonly);
```

Comment:
```suggestion
        if (isDisabledOptionSupported(editor)) {
              setDisabled(editorRef, disabled);
         } else {
	      setReadonly(editorRef, readonly);
        }
```

Would this be clearer?

Reply:
In case of `if editor < 7.6 (isDisabledOptionSupported -> false)` and only `disabled=true`, `readonlyValue` is always `false`, hence the editor will never be switched to `readonly` mode. I could replicate this using storybook. You can try this URL when running locally `http://localhost:6006/?path=/story/editor--disabling&args=channel:7.4`